src/utils/utils.ts

```diff
@@ -53,6 +53,13 @@ export const networkToChainId: Record<string, number> = {
   arbitrum: 42_161,
 }
 
+export const chainIdToNetwork = (): Record<number, string> => {
```

Comment:
Can you just add a little comment about what this is doing? It's not super obvious

Reply:
@alexanderattar sure.